tests/functional/object/mcg/ui/test_s3_vector_ui.py

```
@@ -431,3 +431,188 @@ def teardown():
+            sleep=3,
+            func=lambda: [
+                idx["indexName"]
+                for idx in vector_utils.list_indexes(
+                    s3vectors_client, vectorBucketName=noobaa_bucket_name
+                ).get("indexes", [])
+            ],
+        ):
+            if set(api_indices) == set(remaining_indices):
+                break
+        logger.info(
+            f"Confirmed: {len(remaining_indices)} indices remain after single delete"
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)